MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Observer.java

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.mapboxsdk.maps;
+
+import androidx.annotation.NonNull;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * @brief Observer interface used to subscribe for an \sa Observable events.
+ */
+public abstract class Observer {
+
+  private static AtomicInteger idCounter = new AtomicInteger(0);
+  private final int id;
+
+  public Observer() {
+    id = idCounter.incrementAndGet();
+  }
+
+  /**
+   * @return observer id
+   * @brief Gets observer id
```

Comment:
`@brief` and order: doc before `@return` section

Reply:
fixed